orion/primitives/tadgan.py

```diff
@@ -441,7 +455,18 @@ def score_anomalies(y, y_hat, critic, index, score_window=10, critic_smooth_wind
         rec_error_type)
 
     # Combine the two scores
-    final_scores = np.multiply(critic_scores, res_scores)
+    if comb == "mult":
+        final_scores = np.multiply(critic_scores, res_scores)
+
+    elif comb == "sum":
+        final_scores = lam * (critic_scores - 1) + lam * (res_scores - 1)
```

Comment:
It should be: (1 - lam) * (critic_scores - 1) + lam * (res_scores - 1)

This is convex combination so the coefficients should be non-negative and sum to 1